### PR TITLE
Add on_change callback

### PR DESCRIPTION
**Pull request by agavrilov**

### CODE REVIEW

File: src/views/radio.rs [hunk 1]
```diff
@@ -120,15 +131,25 @@ impl<T> RadioButton<T> {
...
+        }
     }
 
     /// Selects this button, un-selecting any other in the same group.
     ///
     /// Chainable variant.
     pub fn selected(self) -> Self {
-        self.with(Self::select)
+        self.with(|s| {
+            s.select();
```
Comment (gyscos) [inline src/views/radio.rs]: Can you add a comment indicating that we're dropping the potential callback here on purpose?

File: src/views/radio.rs [hunk 2]
```diff
@@ -70,6 +74,13 @@ impl<T> RadioGroup<T> {
     pub fn selection(&self) -> Rc<T> {
         self.state.borrow().selection()
     }
+
+    /// Sets a callback to be used when the selection changes.
+    pub fn set_on_change<F: 'static + Fn(&mut Cursive, &T)>(
```
Comment (gyscos) [inline src/views/radio.rs]: Can you add a chainable `pub fn on_change<F>(self, on_change: F) -> Self` as well?

Comment (agavrilov) [reply src/views/radio.rs]: Done

**Comment by gyscos** [comment]
Thanks again!

**Pull request merged**